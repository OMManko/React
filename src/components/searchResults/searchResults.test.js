import { mount, shallow } from 'enzyme';
import React from 'react';
import SearchResults from './searchResults';
import NoResults from '../noResults/noResults';
import MovieList from '../movieList/movieList';

let component;

describe('<SearchResults />', () => {
    beforeEach(() => {
        component = mount(
            <SearchResults />
        );
    });

    afterEach(() => {
        component.unmount();
    });

    it('should render search results block and match snapshot', () => {
        component = shallow(<SearchResults/>);
        expect(component).toMatchSnapshot();
    });

    it('should render no results block', () => {
        expect(component.find(NoResults).length).toEqual(1);
    });

    it('should render movie list', () => {
        expect(component.find(MovieList).length).toEqual(1);
    });
});
