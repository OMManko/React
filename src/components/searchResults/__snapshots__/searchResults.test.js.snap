// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchResults /> should render search results block and match snapshot 1`] = `ShallowWrapper {}`;
